snippets/card-product.liquid

```diff
@@ -75,7 +75,7 @@
               >
               {% comment %}theme-check-enable ImgLazyLoading{% endcomment %}
 
-              {%- if card_product.media[1] != nil and show_secondary_image -%}
+              {%- if card_product.media[1] != null and show_secondary_image -%}
```

Comment:
Commit: ` hide plus icon when product is soldout`

How do I test a sold out product? 👀 When I make the inventory be 0 for a product, it just removes the product from the complimentary products block.

Is that expected?

Reply:
Yeah it's expected. It filters automatically sold out products. It was the `Bo Ivy - Black` product I could test with as it had only 1 available in stock. So adding it to the cart made it sold out. 
Though I think that might be the product you edited too ? As I don't see it anymore in the complementary products coming up on the puppy product

Reply:
@sofiamatulis The recommendations API actually filters out sold-out products, but you'll have to test when there's only 1 product left.

Reply:
This is a wider thing. Quick add works the same everywhere: [video](https://screenshot.click/21-14-x4o0v-x1oey.mp4)
So that could be something to look at on its own 👍 
There is an existing issue [here](https://github.com/Shopify/dawn/issues/1942)